README.md

```diff
@@ -53,10 +64,53 @@ curl --location --request POST 'http://localhost:3000/bot/chat' \
 }'
 ```
 
-## 未来 TODO
-- `chatGPT`  刷新 token
-- 通过 用户名，密码 初始化 `chatGPT`
+### 微信公众号接入
+
+> 前置条件:
+> - 已注册了微信公众号。https://mp.weixin.qq.com/
+> - 公众号开启了微信云托管服务。https://cloud.weixin.qq.com/cloudrun
+
+#### 新建服务
+
+进入[云托管控制台](https://cloud.weixin.qq.com/cloudrun/console), 点击新建服务，输入服务名称和公网访问，创建新服务即可。
+
+![image](./docs/create-serve.png)
+
+#### 部署发布
+
+部署发布
+你可以通过 fork [GitHub - hua-bang/chatGPT-bot](https://github.com/hua-bang/chatGPT-bot) 这个仓库，新建你的个人仓库(建议设为私有仓库)。
+
+- **选择方式**：绑定 Github 仓库
+- **代码仓库**：fork 后的仓库
+- **分支**：master(如果你不改新分支, 默认 master 就好)。
+- **端口**：3000(如果你不改新分支, 默认 3000 就好)。
+- **注意**： fork 后的代码需要修改 **env** 下的 **OPENAI_API_KEY**。
+
+![image](./docs/publish-server.png)
+
+待部署成功。
+
+#### 配置信息推送
+
+最后一步，是公众号配置信息推送。
+**按下图点击，并打开配置框**：
+- **环境 ID**：当前微信云托管的 id。
+- **服务名称**：上一步创建的服务名称。
+- **PATH**：/mpOfficial/response (如果你不改代码, 默认就是这个)。
+- **推送模式**： 仅支持 JSON。
+
+![image](./docs/config-official.png)
+
+#### 验证程序
+
+找对应的公众号，进行回复测试即可啦。
+
+![image](./docs/chat.jpg)
 
+## 参考
+- [Github-chatgpt-api](https://github.com/transitive-bullshit/chatgpt-api)
+- [completions](https://platform.openai.com/docs/api-reference/completions)
 
 
 
```

Comment:
The code patch is a README file for a ChatGPT-BOT project. It provides instructions on how to use the bot and how to improve it. The README mentions two areas of improvement: 1) making the chat service context-aware, and 2) improving the responsiveness of the WeChat public account reply by switching from passive to active replies. The README also provides instructions on how to use the bot and how to deploy it on WeChat's cloud hosting service. Overall, the code review looks good, and the instructions are clear and concise.